Tests/NatsTests/Integration/ConnectionTests.swift

```diff
@@ -13,6 +13,7 @@
 
 import Logging
 import NIO
+import NatsServer
```

Comment:
should we name this `NatsServerRunner`? (or something similar, wrapper, process etc.)

Reply:
I tried to avoid long name. I'm ok with `NatsServerRunner`. @piotrpio any better names? :)

Reply:
I was tiny bit worried the it is slightly confusing but I doubt we will ever have a NATS server implementation in Swift :-P ..so it's not so bad if it stays as it is as well